pkgs/servers/web-apps/cryptpad/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, buildNpmPackage, fetchFromGitHub, nodejs, nixosTests }:
+
+buildNpmPackage rec {
+  pname = "cryptpad";
+  version = "5.4.0";
+
+  src = fetchFromGitHub {
+    owner = "cryptpad";
+    repo = pname;
```

Comment:
Please don't use this, use a string literal for this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
Fixed